File: ui/visual-debugger/src/main/java/org/visual/debugger/component/VirtualMachineDescriptorListView.java
package org.visual.debugger.component;

import com.sun.tools.attach.VirtualMachineDescriptor;
import javafx.scene.control.ListView;

public class VirtualMachineDescriptorListView extends ListView<VirtualMachineDescriptor> {

  public VirtualMachineDescriptorListView() {
    setCellFactory(new VirtualMachineDescriptorCellFactory());
  }
}

File: visual-model-debugger/src/main/kotlin/org/visual/model/debugger/component/VirtualMachineDescriptorListView.kt
package org.visual.model.debugger.component

import com.sun.tools.attach.VirtualMachineDescriptor
import javafx.scene.control.ListView

class VirtualMachineDescriptorListView : ListView<VirtualMachineDescriptor>() {

    init {
        cellFactory = VirtualMachineDescriptorCellFactory()
    }
}